src/mzn_bench/cli.py

```diff
@@ -207,9 +207,9 @@ def check_statuses(dir: str, pytest_args: Iterable[str]):
     help="Create a row for every problem",
 )
 @click.option(
-    "--avg-time",
-    is_flag=True,
-    help="Show average runtime in the table",
+    "--avg",
+    type=click.Choice(["time", "solveTime", "flatTime"]),
+    help="Show average of the given stat in the table",
```

Comment:
I don't think I know `click` well enough. Does this still allow the option without any time in the table? (That would still be important when using the script for LaTeX exports and such)

Reply:
Yes, in that case it'll be `None`, which is handled in `report_status`